documentation/how-to/authorization.md

```diff
@@ -0,0 +1,262 @@
+# Authorization
+
+> **Note:** Functionality described on this page is not live yet.
+
+Enabling authorization requires that you have an active
+[Charmed Temporal K8s Operator](https://discourse.charmhub.io/t/charmed-temporal-k8s-tutorial-introduction/11777)
+deployed as described in the tutorial. It is recommended that you first go
+through the steps of enabling authentication as outlined
+[here](./authentication.md).
+
+By default the Temporal Server doesn't offer any authorization, but it offers
+the plugin mechanism to add a custom one. We have added an OAuth-based
+[authentication](./authentication.md) using Google Cloud and an authorization
+mechanism that leverages [Google Cloud](https://cloud.google.com) and
+[OpenFGA](https://openfga.dev/). This custom build of the Temporal Server can be
+found [here](https://github.com/canonical/charmed-temporal-image).
+
+## Deploy OpenFGA
+
+To deploy Charmed OpenFGA K8s, you need to run the following command, which will
+fetch the charm from [Charmhub](https://charmhub.io/openfga-k8s) and deploy it
+to your model:
+
+```bash
+juju deploy openfga-k8s --channel edge
```

Comment:
I think this can go to the stable channel, I've been waiting for another team to use it and give feedback before we do so. Let us know if you're having success with it.

Reply:
I agree, I've tested that the charm works well, and I believe the Contracts team has also tested it.

Reply:
I agree, I've tested that the charm works well.